hoodie-client/src/main/java/com/uber/hoodie/io/compact/HoodieRealtimeTableCompactor.java

```diff
@@ -121,6 +133,12 @@
           s.getStat().setTotalLogFiles(scanner.getTotalLogFiles());
           s.getStat().setTotalLogRecords(scanner.getTotalLogRecords());
           s.getStat().setPartitionPath(operation.getPartitionPath());
+          s.getStat().setTotalLogFilesSize((long) operation.getMetrics().get(
+              CompactionStrategy.TOTAL_LOG_FILE_SIZE));
+          s.getStat().setTotalLogBlocks(scanner.getTotalLogBlocks());
+          s.getStat().setTotalCorruptLogBlock(scanner.getTotalCorruptBlocks());
```

Comment:
No `setTotalDataBlocks`?

Reply:
just total log blocks should be enough i think